src/commands/exec.ts

```diff
@@ -57,6 +57,11 @@ export default class Exec extends BaseCommand {
       default: undefined,
       sensitive: false,
     }),
+    replica: Flags.string({
```

Comment:
Can we have this take a pod name or a number?

Sometimes I just want to get the first replica and I don't care about a specific one.

```
architect exec -r 0
```

@TylerAldrich thoughts?

Might not be the right format, but in a CI system I won't know the name of a pod.

Reply:
I do think we should do something like this - we probably need to at least pass the service name because `architect exec -a .. -e .. -r 0` wouldn't work if you have >1 services with >1 replicas.

If you have services called `foo` and `bar` it would be pretty slick IMO to be able to do `-r foo:0` or `-r bar:1` and exec into the correct service+pod

Reply:
We probably could handle `-r 0` if you only have 1 service or only have 1 service with >1 replica because we could deduce which one you mean, and otherwise complain that `-r 0` is too ambiguous - I'm not sure if that adds a ton of complexity but may be the best possible UX because for a lot of simple cases `-r 0` is good enough and you don't even need to write out the service name

Reply:
Sounds good. I'll work on that!

Reply:
I got the `-r foo:0` working but still need to handle `-r 0` for the case where there is only 1 service.